.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "upstream"]
+	path = upstream
+	url = http://bitmath.org/git/mtdev.git
```

Comment:
I would ask for a mirror for this.

Reply:
Yea, could make sense here. Asked. I'll assume replacing this with one is still approved.

Reply:
Yes.